gocd/generated-pipelines/relay-next-us.yaml

```diff
@@ -1,14 +1,14 @@
 format_version: 10
 pipelines:
   deploy-relay-next-us:
-    display_order: 1
+    display_order: 3
```

Comment:
How did this get generated? Does it come from `libs` you updated to?

Reply:
yes it comes from the libs. The order becomes:

1. deploy-<service>
2. rollback-<service>
3. deploy-<service>-<first region>
4. deploy-<service>-<second region> 
5. .....